src/editor/editor.cpp

```diff
@@ -683,7 +690,7 @@ Editor::setup()
     m_enabled = true;
     m_toolbox_widget->update_mouse_icon();
   }
-  
+
```

Comment:
Empty lines at the ends of functions can be removed.

Reply:
my auto formatter did this